test/test_nn.py

```diff
@@ -6119,6 +6119,31 @@ def test_upsamplingBilinear2d(self):
                 input = torch.randn(1, 1, 2, 2, requires_grad=True)
                 gradcheck(lambda x: F.upsample(x, out_size, **kwargs), [input])
 
+    def test_upsamplingBicubic2d(self):
+        # test output against known input
+        in_t = torch.arange(4).view(1, 1, 2, 2).type(torch.FloatTensor)
+        expected_out_t = torch.Tensor(
+            [[[[0.00000, 0.31481, 0.68519, 1.00000],
```

Comment:
with PIL we get something like
```python
array([[-0.2647059 ,  0.03067227,  0.6163866 ,  0.9117647 ],
       [ 0.32605043,  0.62142855,  1.2071428 ,  1.502521  ],
       [ 1.497479  ,  1.7928572 ,  2.3785715 ,  2.6739497 ],
       [ 2.0882354 ,  2.3836133 ,  2.9693277 ,  3.264706  ]],
      dtype=float32)
```
Is that just because of the A factor just above, or maybe the `align_corners`?